cmd/manager/main.go

```diff
@@ -94,14 +94,13 @@ func main() {
 		os.Exit(1)
 	}
 
-	if err = (&controllers.OperatorReconciler{
+	if err = controllers.SetupWithManager(&controllers.OperatorReconciler{
 		Client: mgr.GetClient(),
 		Scheme: mgr.GetScheme(),
 		Resolver: solver.NewDeppySolver(
 			entitysources.NewCatalogdEntitySource(mgr.GetClient()),
-			olm.NewOLMVariableSource(mgr.GetClient()),
-		),
-	}).SetupWithManager(mgr); err != nil {
+			olm.NewOLMVariableSource(mgr.GetClient())),
```

Comment:
Nit: Keep the `solver.NewDeppySolver` closing parenthesis on it's own line. IMO it makes it easier to see where the call stops at a glance